Source/Tools/run_diff.py

```diff
@@ -32,7 +32,7 @@ def run_full_diff(pr_number, token, commit_id):
     diff_vi = os.path.join(tools_directory , "run_diff.vi")
     _logger.debug(f"Launching {diff_vi}.")
 
-    kwargs = ["LabVIEWCLI", "-OperationName", "RunVI", "-VIPath", os.path.normpath(diff_vi)]
+    kwargs = ["LabVIEWCLI", "-OperationName", "RunVI", "-VIPath", os.path.normpath(diff_vi), "-LabVIEWPath", "C:\\Program Files\\National Instruments\\LabVIEW 2020\\LabVIEW.exe"]
```

Comment:
Why is run_diff.py inconsistent with run_vi_analyzer.py? One accepts the LV path as an option and the other hardcodes it.

Reply:
Initially `run_vi_analyzer.py` was expected to run for all LV version but later it was decided to run only one. That's why it is inconsistent. Now, I removed the hard-coded labview path from `run_diff.py` and passed it as an argument same as `run_vi_analyzer.py`.

Reply:
I have made both files consistent in PR [here](https://github.com/ni/measurementlink-labview/pull/509).